src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -16,20 +16,25 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
 
   renderConsignmentsLine(artistsCount) {
     return (
-      <Serif size="3t" color="black60">
+      <Sans size="3t" color="black60">
         Want to sell a work by {artistsCount === 1 ? "this artist" : "these artists"}?{" "}
         <Text style={{ textDecorationLine: "underline" }} onPress={() => this.handleTap("/consign/info")}>
           Consign with Artsy.
         </Text>
-      </Serif>
+      </Sans>
     )
   }
 
   render() {
     const { artwork } = this.props
     const consignableArtistsCount = artwork.artists.filter(artist => artist.is_consignable).length
 
-    return <Box>{!!consignableArtistsCount && this.renderConsignmentsLine(consignableArtistsCount)}</Box>
+    return (
+      <Box>
+        <Spacer mb={2} />
```

Comment:
Would it be possible to add this spacer to the parent component above this component and not add margin directly to this component?

Reply:
We can, but if we want the space to only render when there are links we'd have to add some extra fields and logic to the parent because not all artwork pages have links here.

Reply:
Cool, I'll leave it up to you, generally I think we want to keep from adding margin to components in this way.